Tasks/InstallAppleCertificate/preinstallcert.ts

```diff
@@ -19,15 +19,18 @@ async function run() {
 
         let certPwd: string = tl.getInput('certPwd');
 
-        // get the P12 details - SHA1 hash and common name (CN)
+        // get the P12 SHA1 hash (thumbprint)
         let p12Hash: string = await sign.getP12SHA1Hash(certPath, certPwd);
-        // give user an option to override the CN as a workaround if we can't parse the certificate
+        // get the certificate's common name (CN), e.g. 'iPhone Developer: Chris Sidi (7RZ3N927YF)'
+        // give user an option to override the CN as a workaround if we can't parse the certificate's subject.
         let p12CN: string = tl.getInput('certSigningIdentity', false);
         if (!p12CN) {
             p12CN = await sign.getP12CommonName(certPath, certPwd);
         }
+        // get the private key's name, e.g. 'iOS Developer: Chris Sidi (Chris Sidi)'
+        let privateKeyName = await sign.getP12PrivateKeyName(certPath, certPwd);
```

Comment:
We can add `privateKeyName` as an output variable in the future if it's useful. I'm worried about people confusing it with the certificate's common name (`signingIdentity`), and would like to get some mileage on this code before we add another output variable to our contract.